setup_sut.sh

```diff
@@ -482,7 +482,9 @@ function make_sut() {
             PKG_CONFIG_PATH="$MODULES_DIR/$nettle:$PKG_CONFIG_PATH" ./configure --enable-static --with-guile-site-dir=no --with-included-libtasn1 --with-included-unistring --without-p11-kit --disable-guile --disable-doc
         )
     elif [[ $sut == etinydtls* ]]; then
-        ( cd $sut_dir ; autoconf ; autoheader ; ./configure )
+        # for ETinyDTLS, we need the CC and CPP to be set to absolute paths (to avert Permission Denied when running make)
+        # we also enable address sanitization to prevent segmentation faults, which can take a long time to register and can be triggered in the current implementation
```

Comment:
This comment (about enabling address sanitization) is very cryptic and needs to be improved (or revised).
But we can discuss this issue in person.